python/res/fm/util/template_render.py

```diff
@@ -0,0 +1,79 @@
+gen_kw_export_file = "parameters";
+
+import jinja2
+import json
+import os
+import yaml
+
+#default export file name found in enkf_defaults.hpp
+DEFAULT_GEN_KW_EXPORT_NAME = "parameters.json"
+
+class TemplateRender:
+    @staticmethod
+    def load_data(filename):
+        """Will try to load data from @filename first as yaml, and if that fails,
+        as json. If both fail, a ValueError with both of the error messages will be
+        raised.
+        """
+        with open(filename) as fin:
+            try:
+                return yaml.load(fin)
+            except Exception as yaml_err:
+                pass
+
+            try:
+                return json.load(fin)
+            except Exception as json_err:
+                pass
+
+        err_msg = '%s is neither yaml (err_msg=%s) nor json (err_msg=%s)'
+        raise IOError(err_msg % (filename, str(yaml_err), str(json_err)))
```

Comment:
Please white space separate functions. In general, if a function is inside a class separate them by one line, if outside a class then separate by two newlines..